alcove/query/api.py

```
@@ -167,6 +168,23 @@ def query(inp: QueryIn):
     return _dispatch_query(inp.query, inp.k, mode=inp.mode, collections=inp.collections)
 
 
+@app.get("/api/plugins")
+def plugins_list(
+    type: str | None = Query(default=None, description="Filter by plugin type: extractor, backend, embedder"),
```

Ruff (0.15.13)

[error] Function argument `type` is shadowing a Python builtin

(A002)